frontend/model/contracts/group.js

```diff
@@ -149,6 +150,18 @@ DefineContract({
     groupMembersCount (state, getters) {
       return getters.groupMembersByUsername.length
     },
+    groupMembersPending (state, getters) {
+      const invites = getters.currentGroupState.invites
+      const invitesValid = Object.keys(invites).filter(id => invites[id].status === INVITE_STATUS.VALID && invites[id].creator !== INVITE_INITIAL_CREATOR)
+      const pendingMembers = {}
+      for (const secretId of invitesValid) {
```

Comment:
This is currently doing 3 loops:

1. `Object.keys(invites)`
2. `.filter(...)`
3. `for (const secretId of ...`

Replace 3 loops with a singe loop:

```js
for (const inviteId in invites) {
  let invite = invites[inviteId]
  if (invite.status === INVITE_STATUS.VALID && invite.creator !== INVITE_INITIAL_CREATOR) {
    // stuff below
  }
}
```